### PR TITLE
Add portmap to rpcbind provides

### PR DESCRIPTION
**Pull request by MateuszMalisz**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Add portmap as an explicit `Provides` to enable build of some packages.

###### Change Log  

- Add `portmap` to rpcbind `Provides`

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Pipeline buddy build

### CODE REVIEW

File: SPECS/rpcbind/rpcbind.spec [hunk 1]
```diff
@@ -89,6 +91,9 @@ fi
 %systemd_postun_with_restart rpcbind.service rpcbind.socket
 
 %changelog
+* Wed Sep 07 2022 Mateusz Malisz <mamalisz@microsoft.com> - 1.2.5-5
+- Add portman to provides
```
Comment (oliviacrain) [inline SPECS/rpcbind/rpcbind.spec:95]:
```suggestion
- Add portmap to provides
```

**Review by oliviacrain — CHANGES_REQUESTED** [review]
Change itself looks good, but please retarget against the `main` branch

File: SPECS/rpcbind/rpcbind.spec (hunk 1 above)
Comment (MateuszMalisz) [reply SPECS/rpcbind/rpcbind.spec]: Good catch!

**Review by oliviacrain — APPROVED** [review]
🚀

**Review by jslobodzian — APPROVED** [review]

**Pull request merged**